bundler/lib/bundler/templates/newgem/github/workflows/main.yml.tt

```diff
@@ -18,10 +18,21 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
+<%- if config[:ext] == 'rust' -%>
+    - name: Set up Ruby & Rust
+      uses: oxidize-rb/actions/setup-ruby-and-rust@0d21ce09c5500315bc61815440f8c4211530f413
```

Comment:
Let's not lock it to a hash, otherwise users won't get critical updates. We already do something similar with `ruby/setup-ruby@v1`, since the `v1` tag actually receives updates.

```suggestion
      uses: oxidize-rb/actions/setup-ruby-and-rust@main
```

Reply:
This is not recommended by [OpenSSF](https://securityscorecards.dev/). It is recommended to point to paritcular fixed version (like SHA in here) and triage each update manually (with assitance of dependabot or similar tool).

Anyway we already use tagged actions for now, I would be ok to move to tagged one if released. At some point we will move to particular fixed versions as well for all actions at once.

Reply:
@deivid-rodriguez an idea what to do now? Should we use main tag for now? To promote OpenSSF best practices, we will need to move all actions to point to particular version later anyway.

Reply:
Main sounds good!